ctoec/data-collection
# Add batch select to the roster view

## Background
Added Carbon's batch select to the roster redesign, and also added placeholder UI for roster search and add record

## GitHub Issue
https://github.com/ctoec/data-collection/issues/1358

## Associated PRs
N/A

## Validation Plan
* Have a roster with two or more records
* Select one, see the search bar replaced with the batch actions
* Select two, see the pluralization of "record"
* Deselect all from the toolbar
* Select all from the toolbar

## Automated Testing
- [ ] If applicable, unit tests have been added to cover this changeset.

## Additional Context
N/A
